### ISSUE TITLE
General: some variables are appearing under "Featured" but not under the variable tree

### ISSUE THREAD

**Issue by danicahelb** [VEuPathDB/web-eda#420]
I noticed this for 2 variables in the WASH-b Kenya study:

Community / Study arm 
Community / Randomization block

These variables show up under "Featured variables" but they are not included in the variable tree below. If I try searching for "Study arm", no variables come up

![image](https://user-images.githubusercontent.com/25450900/132877911-e9367a0b-5836-420b-bdca-6b6802c5530b.png)

![image](https://user-images.githubusercontent.com/25450900/132878266-b939652f-e73f-4314-9609-75071fdfefc0.png)

**Comment by dmfalke** [comment]
I took a quick look, and the category that "Cluster study arm" belongs to is hidden. If a node is hidden, I think we are hiding that node and all of its descendant nodes.

**Comment by danicahelb** [comment]
OK, this may have been an issue with annotation property assignments in the .owl

**Comment by danicahelb** [comment]
I don't think there should every be a use case where a featured variable belongs to a node that is being hidden. We should probably flag cases like this so that the outreach team takes another look at the assigned annotations.

**Comment by SheenaTomko** [comment]
Closing the duplicate ticket I made https://github.com/VEuPathDB/EdaNewIssues/issues/149. 

Note that the featured variables for WASH-B Kenya have been updated, so just "Cluster study arm" is a featured variable from the Community node. This shows up correctly in the Featured variables section, but is missing from the main variable tree below, where it should be under Community. All other featured variables from the Household repeated measure, Participant, and Participant repeated measure entities show up correctly in both the Featured variable section and variable tree, and remember that the Household and Participant nodes are also "hidden" so they don't show up duplicated in the variable tree.

![image](https://user-images.githubusercontent.com/54597319/134734686-0e6ebf73-deb9-4723-98b0-cafb5172665c.png)
